docs/guides/drivers/docker-container.md

```diff
@@ -45,29 +48,57 @@ WARNING: No output specified with docker-container driver. Build result will onl
+`docker-container` driver must be explicitly loaded into the local image store.
+Use the `--load` flag:
 
 ```console
-$ docker buildx build --load -t <image> --builder=container .
+$ docker buildx build . --load -t <image> --builder=container
 ...
  => exporting to oci image format                                                                                                      7.7s
  => => exporting layers                                                                                                                4.9s
- => => exporting manifest sha256:4e4ca161fa338be2c303445411900ebbc5fc086153a0b846ac12996960b479d3                                      0.0s 
- => => exporting config sha256:adf3eec768a14b6e183a1010cb96d91155a82fd722a1091440c88f3747f1f53f                                        0.0s 
- => => sending tarball                                                                                                                 2.8s 
- => importing to docker   
+ => => exporting manifest sha256:4e4ca161fa338be2c303445411900ebbc5fc086153a0b846ac12996960b479d3                                      0.0s
+ => => exporting config sha256:adf3eec768a14b6e183a1010cb96d91155a82fd722a1091440c88f3747f1f53f                                        0.0s
+ => => sending tarball                                                                                                                 2.8s
+ => importing to docker
 ```
 
-The image should then be available in the image store:
+The image becomes available in the image store when the build finishes:
 
 ```console
 $ docker image ls
 REPOSITORY                       TAG               IMAGE ID       CREATED             SIZE
 <image>                          latest            adf3eec768a1   2 minutes ago       197MB
 ```
 
+### QEMU
+
+The `docker-container` driver supports using [QEMU](https://www.qemu.org/) (user
+mode) to build non-native platforms. Use the `--platform` flag to specify which
+architectures that you want to build for.
+
+For example, to build a Linux image for `amd64` and `arm64`:
+
+```console
+$ docker buildx build . \
+  --builder=container \
+  --platform=linux/amd64,linux/arm64 \
+  -t <registry>/<image> \
+  --push
+```
+
+> **Warning**
+>
+> QEMU performs full-system emulation of non-native platforms, which is much
+> slower than native builds. Compute-heavy tasks like compilation and
+> compression/decompression will likely take a large performance hit.
+
 ## Further reading
 
-For more information on the docker-container driver, see the [buildx reference](https://docs.docker.com/engine/reference/commandline/buildx_create/#driver).
+For more information on the Docker container driver, see the
+[buildx reference](https://docs.docker.com/engine/reference/commandline/buildx_create/#driver).
+
+If want to explore builders running on a remote server, see the
+[Kubernetes driver](./kubernetes.md) and the [Remote driver](./remote.md).
```

Comment:
Ahhh this isn't exactly true :see_no_evil: `docker-container` "can" do things on a remote server by using a different docker context. I can't find where we document this explicitly anywhere, but we should explain more of the interaction between `buildx` and `docker context` in the `docker.md` page in the future, and we could cross-link to there from here.